{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/forms/label.jinja

```diff
@@ -1,8 +1,32 @@
-{% macro label(field, tag='') %}
+{% from 'forms/attrs.jinja' import form_attrs %}
+
+
+{% macro label(label, tag='label', target='', error=False, class='', attrs={}) %}
     {% if tag %}
         <{{ tag }}
-        class="block text-sm font-medium text-gray-700">{{ field.label }}</{{ tag }}>
+        class="{{class}} {{attrs.pop('class', '')}}  absolute z-10 {% if error %}text-error-focus{% else %}text-gray-400{% endif %}"
+        :class="{{ target }}.length > 0 ? '-translate-y-2 translate-x-2 text-xs bg-white px-1' : 'translate-y-2 translate-x-3'"
+        for={{target}}
+        {{form_attrs(attrs)}}
+        >
+          {{ label }}
+        </{{ tag }}>
     {% else %}
-        {{ field.label }}
+        {{ label }}
+    {% endif %}
+{% endmacro %}
+
+
+{% macro django_label(field, label_text, use_tag, tag, attrs={}) %}
```

Comment:
A comment for the difference between label / django_label would go a long way to improving usability here

Reply:
I could name similar to the others - django_to_label or something?

Reply:
I agree with adding a comment. I think it would be faster for gaining context.